src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-501/rgw-501.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import * as _ from 'lodash';
+
+@Component({
+  selector: 'cd-rgw-501',
+  templateUrl: './rgw-501.component.html',
+  styleUrls: ['./rgw-501.component.scss']
+})
+export class Rgw501Component implements OnInit, OnDestroy {
+
+  message = 'The RGW service is not configured.';
+  routeParamsSubscribe: any;
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit() {
+    this.routeParamsSubscribe = this.route.params
+      .subscribe((params: {message: null|string}) => {
```

Comment:
Depending on the solution you choose from my previous comment, you may have to remote the `null` type here.